src/main/java/com/teamrocket/tms/services/TaskServiceImpl.java

```diff
@@ -27,4 +30,24 @@ public TaskDTO createTask(TaskDTO taskDTO) {
 
         return modelMapper.map(savedTaskEntity, TaskDTO.class);
     }
+
+    @Override
+    public List<TaskDTO> getAllTasks() {
+        List<Task> taskList = taskRepository.findAll();
+        List<TaskDTO> taskDTOList = new ArrayList<>();
+        taskList.stream()
+                .forEach(element -> taskDTOList.add(modelMapper.map(element, TaskDTO.class)));
+        log.info("Task list retrieved.");
+
+        return taskDTOList;
+    }
+
+    @Override
+    public TaskDTO getTaskById(Long id) {
+        Task task = taskRepository.findById(id)
+                .orElseThrow(() -> new RuntimeException("task.not.found"));
```

Comment:
Here maybe we can throw a custom exception? Something like "TaskNotFoundException"?